### DISCUSSION TITLE
programming/2012/04/06/icopyhook

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#229] [Comments]
# programming/2012/04/06/icopyhook

ICopyHook은 윈도에서 제공하는 COM 인터페이스이다. 이 인터페이스를 구현해서 시스템에 등록시키면 쉘을 통한 파일 오퍼레이션이 발생할 때 내가 설치해 놓은 코드가 실행되도록 할 수 있다.

https://jeho.page/programming/2012/04/06/icopyhook.html

**Comment by yuninsik** [comment]
링크해갑니다~

**Reply by BenjaminKim** [reply]
너무 오래된 글이네요. 10년이 지나 다시 읽어보니 좀 부끄럽습니다. ㅎ
